charts/node-analyzer/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: node-analyzer
 description: Sysdig Node Analyzer
 # currently matching Sysdig's appVersion 1.14.34
-version: 1.29.0
+version: 1.29.1
```

Comment:
As this is a new feature, please bump a minor.

Reply:
Didn't think about it but you're right @mavimo , thanks for spotting this! :pray: